api/python/quilt3/api.py

```diff
@@ -164,38 +166,23 @@ def _disable_telemetry():
 
 
 @ApiTelemetry("api.search")
-def search(query, limit=10):
+def search(query: T.Union[str, dict], limit: int = 10) -> T.List[dict]:
     """
     Execute a search against the configured search endpoint.
 
     Args:
-        query (str): query string to search
-        limit (number): maximum number of results to return. Defaults to 10
+        query: query string to search if passed as `str`, ES query body if passed as `dict`
+        limit: maximum number of results to return. Defaults to 10
 
     Query Syntax:
-        [simple query string query](
-            https://www.elastic.co/guide/en/elasticsearch/reference/6.8/query-dsl-simple-query-string-query.html)
-
+        [Query String Query](
+            https://www.elastic.co/guide/en/elasticsearch/reference/6.8/query-dsl-query-string-query.html)
+        [Query DSL](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/query-dsl.html)
```

Comment:
is it actually 6.8 or 6.7?

Reply:
we use 6.7 but I don't think it's any different to 6.8 here